Frontend/src/components/EmbedDashboardLayout/EmbedDashboardLayout.js

```diff
@@ -119,19 +100,21 @@ const EmbedDashboardLayout = () => {
       "data_block_acs_bigquery::acs_census_overview"
     )
 
-      // adds the iframe to the DOM as a child of a specific element
+      // Adds the iframe to the DOM as a child of a specific element
       .appendTo(el)
-      // this instructs the SDK to point to the /dashboards-next/ version
+      // Instructs the SDK to point to the /dashboards-next/ version
       .withNext()
-      // this is an event listener for when the dashboard is finished running
-      .on("dashboard:run:complete", initializeDashboardOptions)
-      // this line performs the call to the auth service to get the iframe's src='' url, places it in the iframe and the client performs the request to Looker
+      // Set event listener to store the dashboard's initial options when the dashboard loads
+      .on("dashboard:loaded", initializeDashboardOptions)
+      // Set an event listener to update our state with the dashboard's latest options when the dashboard finishes its queries
+      .on("dashboard:run:complete", event => setDashboardOptions(event.dashboard.options))
+      // Performs the call to the auth service to get the iframe's src='' url, places it in the iframe and the client performs the request to Looker
       .build()
-      // this establishes event communication between the iframe and parent page
+      // Establishes event communication between the iframe and parent page
       .connect()
-      // this line sets up the dashboard after the building is complete
+      // Sets up the dashboard after its built
```

Comment:
```suggestion
      // Sets up the dashboard after it's built
```

nit

Reply:
Updated. Wonder if a vscode extension exists for comment grammar